src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, serif;
+
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-buy-color: #00acdc;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  height: 408px;
```

Comment:
Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.